horton/gbasis/ints.cpp

```diff
@@ -235,8 +236,18 @@ void GB2NuclearAttractionIntegral::add(double coeff, double alpha0, double alpha
 
         // Fill the work array with the Boys function values
         arg = gamma*(pc[0]*pc[0] + pc[1]*pc[1] + pc[2]*pc[2]);
-        for (long nu=abs(shell_type0)+abs(shell_type1); nu>=0; nu--) {
-            work_boys[nu] = boys_function(nu, arg);
+        // for (long nu=abs(shell_type0)+abs(shell_type1); nu>=0; nu--) {
+        //    work_boys[nu] = boys_function(nu, arg);
+        //}
+        /*
+           Laplace transform of the potential
+        */
+
+        int mmax = abs(shell_type0) + abs(shell_type1);
+        double kernel[2*max_shell_type+1];
```

Comment:
It seems that there is no need to define and allocate this array. Can you try the following instead?
- Remove this line.
- Use `work_boys` as last argument in the call to `laplace_of_potential`
- Remove the loop below `laplace_of_potential`.